src/cli.py

```diff
@@ -37,12 +37,24 @@ def check(protostar: bool, directory: str, files: List[str]):
         files = get_contracts_from_protostar(protostar_path)
     sys.exit(check_files(directory, files))
 
-
+# this command may be run with:
+# python src/cli.py order-imports -f test/main_imports_test.cairo -i starkware -i openzeppelin
+# python src/cli.py order-imports -d test/ -i starkware -i openzeppelin
 @click.command()
+@click.option('--directory', '-d', help="Directory with cairo files to format")
 @click.option("--files", '-f', multiple=True, default=[], help="File paths")
-def order_imports(directory: str, files: List[str]):
+@click.option("--imports", '-i', multiple=True, default=["starkware", "openzeppelin"], help="Imports order")
+def order_imports(directory: str, files: List[str], imports: List[str]):
+    files_to_order = []
+    if len(directory) > 0:
```

Comment:
If the directory is unspecified, this will crash the program. Consider using `if directory :` instead

Reply:
thanks